docs/deployed-contracts/hoodi.md

```diff
@@ -57,6 +57,21 @@ Currently, the main operational and maintained protocol testnet is being migrate
 - Gate Seal Factory: [`0xA402349F560D45310D301E92B1AA4DeCABe147B3`](https://hoodi.etherscan.io/address/0xA402349F560D45310D301E92B1AA4DeCABe147B3)
 - Gate Seal: [`0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C`](https://hoodi.etherscan.io/address/0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C)
 
+### Dual Governance
+
+- Emergency Protected Timelock: [`0x0A5E22782C0Bd4AddF10D771f0bF0406B038282d`](https://hoodi.etherscan.io/address/0x0A5E22782C0Bd4AddF10D771f0bF0406B038282d) (proposed)
+- Admin Executor: [`0x0eCc17597D292271836691358B22340b78F3035B`](https://hoodi.etherscan.io/address/0x0eCc17597D292271836691358B22340b78F3035B) (proposed)
+- Dual Governance: [`0x4d12b9f6aCAB54FF6a3a776BA3b8724D9B77845F`](https://hoodi.etherscan.io/address/0x4d12b9f6aCAB54FF6a3a776BA3b8724D9B77845F) (proposed)
+- Dual Governance Config Provider: [`0x2b685e6fB288bBb7A82533BAfb679FfDF6E5bb33`](https://hoodi.etherscan.io/address/0x2b685e6fB288bBb7A82533BAfb679FfDF6E5bb33) (proposed)
+- Emergency Timelocked Governance: [`0xF1Fbb8360C19830228260Ed5Bb432476A9d4897a`](https://hoodi.etherscan.io/address/0xF1Fbb8360C19830228260Ed5Bb432476A9d4897a) (proposed)
```

Comment:
Here should be "producation-ready" emergency governance `0x49B3512c44891bef83F8967d075121Bd1b07a01B`

Reply:
The address `0xF1Fbb8360C19830228260Ed5Bb432476A9d4897a` should be included in the section listing the contracts used for the DG launch.